velox/exec/tests/utils/QueryAssertions.h

```diff
@@ -201,6 +201,11 @@ bool assertEqualResults(
     const std::vector<RowVectorPtr>& expected,
     const std::vector<RowVectorPtr>& actual);
 
+/// Ensure both plans have the same results.
+bool assertEqualResults(
```

Comment:
Maybe extract this change into a separate PR.

Reply:
Sure. https://github.com/facebookincubator/velox/pull/8134